sdk/src/main/java/com/mapbox/maps/NativeMapImpl.kt

```diff
@@ -508,4 +508,8 @@ internal class NativeMapImpl(private val map: MapInterface) :
   override fun unsubscribe(observer: Observer) {
     map.unsubscribe(observer)
   }
+
+  override fun setMapProjection(value: Value) = map.setMapProjection(value)
```

Comment:
Add type explicitly

Reply:
as mentioned in description, work will be done in #667